zha/application/platforms/sensor/__init__.py

```diff
@@ -173,6 +174,13 @@ def __init__(
     ) -> None:
         """Init this sensor."""
         self._cluster_handler: ClusterHandler = cluster_handlers[0]
+        self._attr_def: foundation.ZCLAttributeDef | None = None
+
+        if self._attribute_name is not None:
```

Comment:
If you wanted to, could use `if (attr := self._attribute_name) is not None:` to just access `attr` for the name below, but this is obviously also fine.